class/deeplink-schema-match.ts

```
@@ -34,172 +34,192 @@ class DeeplinkSchemaMatch {
-          if (!file || !this.hasNeededJsonKeysForMultiSigSharing(file)) {
-            return;
-          }
+    }
+
+    // Handle possibly cosigner files
+    if (DeeplinkSchemaMatch.isPossiblyCosignerFile(event.url)) {
+      try {
+        const file = await readFileOutsideSandbox(decodeURI(event.url));
+        // Checks whether the necessary json keys are present in order to set a cosigner,
+        // doesn't validate the values this happens later
+        if (file && this.hasNeededJsonKeysForMultiSigSharing(file)) {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)